src/uwtools/config/formats/fieldtable.py

```diff
@@ -12,6 +12,31 @@ class FieldTableConfig(YAMLConfig):
     an input YAML file.
     """
 
+    # Private methods
+
+    @classmethod
+    def _dict_to_str(cls, cfg: dict) -> str:
+        """
+        Returns the field-table representation of the given dict.
+
+        :param cfg: A dict object.
+        """
+        lines = []
+        for field, settings in cfg.items():
+            lines.append(f' "TRACER", "atmos_mod", "{field}"')
+            for key, value in settings.items():
+                if isinstance(value, dict):
+                    method_string = f'{" ":7}"{key}", "{value.pop("name")}"'
+                    # All control vars go into one set of quotes.
+                    control_vars = [f"{method}={val}" for method, val in value.items()]
+                    # Whitespace after the comma matters.
+                    lines.append(f'{method_string}, "{", ".join(control_vars)}"')
+                else:
+                    # Formatting of variable spacing dependent on key length.
+                    lines.append(f'{" ":11}"{key}", "{value}"')
+            lines[-1] += " /"
+        return "\n".join(lines)
```

Comment:
This pattern repeats for each of the `Config` subclasses that follow: Logic from the `dump_dict()` method is factored out to `_dict_to_str()`, which `dump_dict()` then calls. This also allows `__repr__()` to call `_dict_to_str()` to get a format-native representation.